### DISCUSSION TITLE
2D Gaussian convolution in pde of Deepxde

### DISCUSSION THREAD

**Discussion by Hari-IITian** [lululxvi/deepxde#1726] [Q&A]
I want to use 2d Gaussian convolution in the definition of pde of deepxde. How to do that? 
let us suppose I want to solve 
u_t-(Gu)=0 ,  
here Gu represents 2d spatial Gaussian  convolution of u.

**Comment by lululxvi** [comment]
See https://doi.org/10.1137/19M1274067 Section 2.6
